### PR TITLE
106: Add CC0 license

### PR DESCRIPTION
**Pull request by jamesobrooks**
# CC0 License

Add CC0 license to LICENSE.md
Add CONTRIBUTING.md with disclaimer of waiver of rights pursuant to the CC0 license

## LICENSE.md Text

The text used comes from a couple of 18f sources, combining a couple of things from [the 18f.gsa.gov license](https://github.com/18F/18f.gsa.gov/blob/main/LICENSE.md) and the markdown from [the 18F Open Source Policy](https://github.com/18F/open-source-policy/blob/master/LICENSE.md).

The summary can be found on [the Creative Commons](https://creativecommons.org/choose/zero/) website.

## CONTRIBUTING.md

As was done on [the 18f.gsa.gov license](https://github.com/18F/18f.gsa.gov/blob/main/LICENSE.md), we've added a disclaimer to the contribution guidelines to inform contributors of their waiver of rights to contributions.

### Contact Email

USTP is requesting an email address be created for the product. Once that email is in place, we should add something like the following to this file:

```
If you're unsure about anything, just [send us an email](mailto:email@doj.gov) with your question — or submit the issue or pull request anyway.
```

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## Public contributions
+
+We're so glad you're thinking about contributing to a DOJ USTP open source project! If you're unsure about anything, just [send us an email](mailto:18f@gsa.gov) with your question — or submit the issue or pull request anyway. The worst that can happen is you'll be politely asked to change something. We love all friendly contributions.
```
[the remark this answers is not in the document]
Comment (jamesobrooks) [reply CONTRIBUTING.md:3]: Also, where the 18F repo had `We're so glad you're thinking about contributing to an 18F open source project!`, I changed `18F` to `DOJ USTP`. Not sure if that's how it should read or not.

File: LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# License
+
+As a work of the [United States government](https://www.usa.gov/), this project is in the public domain within the United States of America.
+
+Additionally, we waive copyright and related rights in the work worldwide through the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/publicdomain/zero/1.0/).
```
Comment (jamesobrooks) [inline LICENSE.md:5]: @jayaneti1 I wasn't certain about this, but it seems to be on all the 18F repos. Does this worldwide waiver apply?

Comment (fmaddenflx) [reply LICENSE.md:5]: Good question.  Otherwise looks good.

Comment (governmentSponsored) [reply LICENSE.md:5]: I believe a worldwide waiver is appropriate

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+## Public contributions
+
+We're so glad you're thinking about contributing to a DOJ USTP open source project! If you feel you have something to add to the project, please submit a pull request. The worst that can happen is you'll be politely asked to change something. We love all friendly contributions.
+
+We want to ensure a welcoming environment for all of our projects. Our staff follow the [18F Code of Conduct](https://github.com/18F/code-of-conduct/blob/master/code-of-conduct.md) and all contributors should do the same.
```
Comment (governmentSponsored) [inline CONTRIBUTING.md:5]: Think this link needs some work. Not sure if we want to link to DOJ code of conduct or use 18F's. Additionally, the link used here goes to an archived GH repo, which in turn points to this instead: https://handbook.tts.gsa.gov/about-us/code-of-conduct/.

Comment (governmentSponsored) [inline CONTRIBUTING.md:5]: Think this link needs some work. Not sure if we want to link to DOJ code of conduct or use 18F's. Additionally, the link used here goes to an archived GH repo, which in turn points to this instead: https://handbook.tts.gsa.gov/about-us/code-of-conduct/.

Comment (governmentSponsored) [inline CONTRIBUTING.md:5]: Think this link needs some work. Not sure if we want to link to DOJ code of conduct or use 18F's. Additionally, the link used here goes to an archived GH repo, which in turn points to this instead: https://handbook.tts.gsa.gov/about-us/code-of-conduct/.

Comment (governmentSponsored) [inline CONTRIBUTING.md:5]: Think this link needs some work. Not sure if we want to link to DOJ code of conduct or use 18F's. Additionally, the link used here goes to an archived GH repo, which in turn points to this instead: https://handbook.tts.gsa.gov/about-us/code-of-conduct/.

**Review by governmentSponsored — APPROVED** [review]
IMO changes look good.

**Pull request merged**